### PR TITLE
Add spyder 2.3.8

### PR DESCRIPTION
**Pull request by malfatti**
Just copied spyder.2.3.5.2.ebuild and adapted to spyder2.3.8.

### CODE REVIEW

**Comment by idella** [comment]
I don't know if this is my code perpetuated on a bad office day but  why is
use doc && dodoc -r doc/html/ 
not:   use doc && local HTML_DOCS=(  doc/html/. )

in fact should could be

```ebuild
python_install_all() {
+	use doc && local HTML_DOCS=(  doc/html/. )
+	distutils-r1_python_install_all
+	doicon spyderlib/images/spyder.svg
+	make_desktop_entry spyder Spyder spyder "Development;IDE"
```

**Comment by malfatti** [comment]
Ahh, I see! Sorry about that, I just copied the previous ebuild and adapted it, and when I ran `ebuild spider-2.3.8.ebuild manifest clean merge` it installed without problems. I'll add your code and the new IUSE flag. Thanks a lot!

**Comment by idella** [comment]
@gentoo/python

**Comment by idella** [comment]
Next, I believe it was I the last to have lat bumped it:
spyder-2.3.5.2.ebuild has  PYTHON_COMPAT=( python{2_7,3_3,3_4} )
this bumped version has   PYTHON_COMPAT=( python{2_7,3_4} )
So; it's current policy to retain python3_3 in packages that support it until 3_3 is formerly deprecated or dropped.  You also ought add support to  python3.5

Sorry should have provided these in one hit

**Comment by idella** [comment]
Also have you submitted using repoman commit or git commit ?

**Comment by malfatti** [comment]
Indeed, I thought that python3.3 was already deprecated, sorry about that :) I'll add support to both versions. I'm using git commit, btw.

**Comment by floppym** [comment]
This pull request is kind of messy. Please squash this into one or two clean commits.

Also, your commit messages are not great. The summary lines are missing the cat/pkg: for one.

**Comment by malfatti** [comment]
I'm sorry for the mess, I'm new to github. I tried to figure out how to correct the code without commiting all the time, but I couldn't. Should I close this and open a new pull request?

**Comment by floppym** [comment]
I would suggest an interactive rebase, followed by `git push --force`.

https://help.github.com/articles/about-git-rebase/

**Comment by malfatti** [comment]
Oh, sorry, forgot to squash :)

File: dev-python/spyder/spyder-2.3.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+#
+EAPI=5
```
Comment (jlec) [inline dev-python/spyder/spyder-2.3.8.ebuild]: Why not ```EAPI=6```?

File: dev-python/spyder/spyder-2.3.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc ipython matplotlib numpy pep8 +pyflakes pylint qtconsole +rope scipy sphinx"
+
+# rope requires no version bordering since all are >= miniumum version
+RDEPEND="
+	|| ( dev-python/PyQt4[${PYTHON_USEDEP},svg,webkit]
+		 dev-python/pyside[${PYTHON_USEDEP},svg,webkit] )
```
Comment (jlec) [inline dev-python/spyder/spyder-2.3.8.ebuild]: Extra whitespace, which repoman should have warned you about. Please always use repoman.

File: dev-python/spyder/spyder-2.3.8.ebuild (hunk 1 above)
Comment (malfatti) [inline dev-python/spyder/spyder-2.3.8.ebuild]: I thought EAPI should be defined from 0 to 5 ([https://wiki.gentoo.org/wiki/EAPI](url))... Should I change it do 6? What improvements would this bring?

File: dev-python/spyder/spyder-2.3.8.ebuild (hunk 2 above)
Comment (malfatti) [inline dev-python/spyder/spyder-2.3.8.ebuild]: I kept them the way they were in the previous ebuild (spyder-2.3.5.2). Should I remove lines 6, 8, 10, 14 and 19?

Comment (jlec) [inline dev-python/spyder/spyder-2.3.8.ebuild]: No, don't remove the empty lines. Line 23 has ```\t ```in front. drop the space.

File: dev-python/spyder/spyder-2.3.8.ebuild (hunk 1 above)
Comment (jlec) [inline dev-python/spyder/spyder-2.3.8.ebuild]: Search for EAPI=6 in [PMS](https://projects.gentoo.org/pms/6/pms.html). One important change is, that patching is now natively support by the PM and there is no need to use the eutils eclass anymore.

**Comment by monsieurp** [comment]
ping @malfatti?

**Comment by monsieurp** [comment]
OK. https://github.com/malfatti/gentoo/commit/eb95787f46aeb50b67ffc23498ad5509be1a4147 needs rebasing and editing as well. Then I guess it's ready to go.

**Comment by malfatti** [comment]
Oops, sorry :) Fixing this...

**Comment by malfatti** [comment]
Thank you all guys, for all the guidance :)

**Pull request merged**